pkg/providers/oracle/model/objectstore/objectstore.go

```diff
@@ -1,25 +1,27 @@
 package objectstore
 
 import (
-	pipelineAuth "github.com/banzaicloud/pipeline/auth"
+	"github.com/banzaicloud/pipeline/auth"
 )
 
 // TableName constants
 const (
-	ManagedOracleBucketTableName = "managed_oracle_buckets"
+	BucketsTableName = "oracle_buckets"
```

Comment:
Not sure if this is necessary...the table name is not used at any other place

Reply:
Even so, I think it is useful to have the used constants on top, but it should be unexported.